### DISCUSSION TITLE
Any plans to support the built-in Node.js test runner?

### DISCUSSION THREAD

**Discussion by EricCrosson** [dubzzz/fast-check#3401] [Q&A] [answered]
Hi! I love the consolidation of the different test harnesses (ava, mocha, jest) into the monorepo lately! It seems like this is reducing maintenance overhead. Are there any plans to add the built-in Node.js test runner[^1] to the list of supported test frameworks?

[^1]: https://nodejs.org/api/test.html#test-runner

**Comment by dubzzz** [comment, marked as answer]
Hi!

No plan to add it short term so far. By default, fast-check already support any test runner without extra packages needed.

The extra packages `@fast-check/ava` or `@fast-check/jest` have been created to ensure an even deeper integration with these tests frameworks as they provide deep functionalities that can be connected once for everyone: use the seed of jest, possibly better watch...

**Comment by EricCrosson** [comment]
>  By default, fast-check already support any test runner without extra packages needed.

Aha! I was overthinking it :grin: I'll have to give this a shot!

Thank you for the tip :pray:
